SlevomatCodingStandard/Helpers/ReferencedNameHelper.php

```diff
@@ -92,11 +92,14 @@ private static function createAllReferencedNames(PHP_CodeSniffer_File $phpcsFile
 						], true)
 					) {
 						if ($tokens[$previousTokenBeforeStartPointer]['code'] === T_COMMA) {
-							$implementsPointer = TokenHelper::findPreviousExcluding(
-								$phpcsFile,
-								array_merge(TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),
-								$previousTokenBeforeStartPointer - 1
-							);
+							$implementsPointer = $previousTokenBeforeStartPointer;
+							do {
+								$implementsPointer = TokenHelper::findPreviousExcluding(
+									$phpcsFile,
+									array_merge(TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),
```

Comment:
Nešlo by to jen přes `array_merge([T_COMMA], TokenHelper::$nameTokenCodes, TokenHelper::$ineffectiveTokenCodes),` ?